.github/workflows/ci.yaml

```diff
@@ -62,5 +62,6 @@ jobs:
           SNOWFLAKE_ACCOUNT: gp21411
           SNOWFLAKE_REGION: us-east-1
           SNOWFLAKE_ROLE: TRANSFORMER
+          GCP_CREDENTIALS: ${{ secrets.GCP_CREDENTIALS }}
```

Comment:
We should make this `AIRFLOW__ASTRO__GCP_CREDENTIALS` s.t. people can put this in their `airflow.cfg

Reply:
Sounds great will update it.

Reply:
I think we need to replace the GOOGLE_APPLICATION_CREDENTIALS with AIRFLOW__ASTRO__GCP_CREDENTIALS, since env var GOOGLE_APPLICATION_CREDENTIALS is used while creating sessions in the code and GCP_CREDENTIALS is just used as a workaround for the json file content.